package/lib-edgetpu/.cm/meta.json

```diff
@@ -0,0 +1,49 @@
+{
+  "customize": {
+    "extra_dir": "",
+    "install_env": {
+      "PACKAGE_URL": "https://packages.cloud.google.com/apt/pool/",
+      "PACKAGE_WGET": "YES",
+      "PACKAGE_WGET_EXTRA": "--no-cookies"
+    },
+    "version": "with-variations"
+  },
```

Comment:
I suggest to add `"no_ver_in_suggested_path": "yes",` as [here](https://github.com/ctuning/ck-openvino/blob/master/package/lib-openvino/.cm/meta.json#L24): otherwise, the version gets repeated twice as in `/home/maria/CK-TOOLS/lib-edgetpu-14.1-v14.1_max-linux-64` - once for the version number, and again as the variation name.

Reply:
Ok. Done.